packages/semantic-config/release.config.cjs

```diff
@@ -1,11 +1,22 @@
-const { pluginOptions, releaseConfig } = require('../../release.config.cjs')
+/* eslint-disable import/order */
```

Comment:
If we do this _and_ put the proper defaults in for `@jeromefitz/semantic` we can trim down every single `release.config` considerably as this package as the only one not published to `npm`.

Then going forward any _new_ package that is not going to `npm` would have a one-off `release.config` that passes `enableNpm: false` instead of passing default configuration in every single package like we are now.